github-sample/.github/workflows/IaC_container_deploy.yml

```diff
@@ -0,0 +1,85 @@
+# Docs for the Azure Web Apps Deploy action: https://github.com/azure/functions-action
+# More GitHub Actions for Azure: https://github.com/Azure/actions
+
+name: (container) Logic App Infrastructure as Code Pipeline. 
+
+on:
+  - workflow_dispatch 
+
+env:
+  RG_LA: 'your-la-rg'
+  RG_CON: 'your-con-rg'
+  IMAGE_NAME: 'your-cont-image'
+  IMAGE_TAG: 'your-tag-no'
+
+jobs:
+  deploy-all: 
+    runs-on: ubuntu-latest
+    steps:
+                      
+    # Authentication
+    # Set up the following secrets in your repository: AZURE_CREDENTIALS
+    # For details on usage of secrets, please refer https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets
+    - name: Azure Login
+      uses: azure/login@v1
+      with:
+        creds: ${{ secrets.AZURE_CREDENTIALS  }}
+        enable-AzPSSession: true 
+
+    # Checkout
+    - name: Checkout
+      uses: actions/checkout@v1
+
+    # Deployment of template  
+    - name: Deploy Container Resources
+      uses: Azure/arm-deploy@v1.0.1
+      id: acrdeploy
+      with:
+        # You can change these environment variables for your configuration:   AZURE_SUBSCRIPTION_ID, AZURE_RESOURCE_GROUP
+        scope: resourcegroup  
+        subscriptionId: ${{ secrets.AZURE_SUB }}
+        resourceGroupName:  ${{ env.RG_LA }}
+        template: ARM/acr-template.json  # Set this to the location of your template file
+        parameters: ARM/acr-parameters.json # Set this to the location of your parameters file
+        deploymentMode: Incremental
+   
+    # Deployment of template    
+    - name: Deploy Connector Resources 
+      id: condeploy
+      uses: Azure/arm-deploy@v1.0.1
+      with:
+        scope: resourcegroup  
+        subscriptionId: ${{ secrets.AZURE_SUB }}
+        resourceGroupName: ${{ env.RG_CON }}
+        template: ARM/connectors-template.json  # Set this to the location of your template file
+        parameters: ARM/connectors-parameters.json logicAppSystemAssignedIdentityTenantId=${{ steps.acrdeploy.outputs.logicAppSystemAssignedIdentityTenantId }} logicAppSystemAssignedIdentityObjectId=${{ steps.acrdeploy.outputs.logicAppSystemAssignedIdentityObjectId }}
+        deploymentMode: Incremental
+
+    #build container image
```

Comment:
I think to keep this consistent to the non-container IaC deployment, should we get rid of the build, push  & deploy steps